src/applications/representative-appoint/tests/pages/representative/selectAccreditedRepresentative.unit.spec.jsx

```diff
@@ -1,68 +1,34 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import { expect } from 'chai';
-import {
-  render,
-  waitFor,
-  fireEvent,
-  getByTestId,
-  cleanup,
-} from '@testing-library/react';
-import sinon from 'sinon';
-import { $ } from 'platform/forms-system/src/js/utilities/ui';
-import { SelectAccreditedRepresentative } from '../../../components/SelectAccreditedRepresentative';
-import * as api from '../../../api/fetchRepStatus';
-import * as searchAPI from '../../../api/fetchRepresentatives';
+import { render } from '@testing-library/react';
+import SelectedAccreditedRepresentativeReview from '../../../components/SelectAccreditedRepresentativeReview';
 import repResults from '../../fixtures/data/representative-results.json';
-import * as reviewPageHook from '../../../hooks/useReviewPage';
+import orgResult from '../../fixtures/data/organization-type.json';
 
-describe('<SelectAccreditedRepresentative>', () => {
-  const getProps = ({
-    currentRep = undefined,
-    query = undefined,
-    results = repResults,
-    submitted = false,
-  } = {}) => {
+describe('<SelectAccreditedRepresentativeReview>', () => {
```

Comment:
Oh, the underlying component changed in this spec. Maybe it wasn't from a merge conflict.